gem5-configs/configs-boot-tests/run_exit.py

```diff
@@ -63,10 +63,11 @@
     # create the system we are going to simulate
     if mem_sys == "classic":
         system = MySystem(kernel, disk, cpu_type, num_cpus, opts)
-    elif mem_sys == "MI_example" or "MESI_Two_Level":
+    elif mem_sys == "MI_example" or "MESI_Two_Level" or "MOESI_CMP_directory":
```

Comment:
This is getting a little verbose how about:

```suggestion
    elif mem_sys in ruby_protocols:
```

And somewhere else (in `ruby_system.py`?) have 
```
ruby_protocols = [ "MI_example", "MESI_Two_Level", "MOESI_CMP_directory"]
```